components/Api/Stability/__test__/index.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { render } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
+import Stability from '..';
+
+describe('Stability', () => {
```

Comment:
Can you remove these unit tests? We now use snapshot testing provided by Storybook, @Harkunwar could you write here if any other step needs to be taken 👀

Reply:
Done!